src/hub.js

```diff
@@ -414,6 +415,7 @@ async function updateEnvironmentForHub(hub, entryManager) {
       },
       { once: true }
     );
+    environmentEl.addEventListener("model-error", sceneErrorHandler, { once: true });
```

Comment:
I decided against doing this for scene changes in https://github.com/mozilla/hubs/pull/1981

Reply:
Hmm. I don't think exiting is a very graceful outcome, but it seems important to let the user know about it. How about I only add this event listener if you have not yet entered the scene? I am concerned about the case of having a scene change right when you try to load the page.

Reply:
Ah, yeah, makes sense to catch this error if you haven't entered the scene